memphis/partition_generator.py

```diff
@@ -0,0 +1,12 @@
+
+
+class PartitionGenerator:
```

Comment:
Can you add some documentation indicating what this class does and how it should be used?